reportportal/README.md

```diff
@@ -80,3 +80,82 @@ mongodb:
     address: mongodb://mongodb.default.svc.cluster.local
     port: 27017
 ```
+
+### Installation notes
+1. Make sure you have Kubernetes up and running
+2. Reportportal requires installed [mongodb](https://github.com/helm/charts/tree/master/stable/mongodb) and [elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch) to run. Required versions of helm charts are described in requirements.yaml
+If you don't have your own mongodb and elasticsearch you can install them from helm charts. 
```

Comment:
If you don't have your own mongodb and elasticsearch instances, they can be installed from official helm charts.